README.md

```diff
@@ -72,23 +69,87 @@ respond "Hello World!"
+        configuration {
+            # Required value
+            ValueDir <string>
+
+            # Optional
+            SyncWrites <bool>
+            NumVersionsToKeep <int>
+            ReadOnly <bool>
+            Compression <int>
+            InMemory <bool>
+            MetricsEnabled <bool>
+            MemTableSize <int>
+            BaseTableSize <int>
+            BaseLevelSize <int>
+            LevelSizeMultiplier <int>
+            TableSizeMultiplier <int>
+            MaxLevels <int>
+            VLogPercentile <float>
+            ValueThreshold <int>
+            NumMemtables <int>
+            BlockSize <int>
+            BloomFalsePositive <float>
+            BlockCacheSize <int>
+            IndexCacheSize <int>
+            NumLevelZeroTables <int>
+            NumLevelZeroTablesStall <int>
+            ValueLogFileSize <int>
+            ValueLogMaxEntries <int>
+            NumCompactors <int>
+            CompactL0OnClose <bool>
+            LmaxCompaction <bool>
+            ZSTDCompressionLevel <int>
+            VerifyValueChecksum <bool>
+            EncryptionKey <string>
+            EncryptionKey <Duration>
+            BypassLockGuard <bool>
+            ChecksumVerificationMode <int>
+            DetectConflicts <bool>
+            NamespaceOffset <int>
+        }
+    }
 }
 
 cache @match2 {
     ttl 50s
+    badger {
+        path /tmp/badger/second-match
+        configuration {
+            ValueDir match2
+            ValueLogFileSize 16777216
+            MemTableSize 4194304
+            ValueThreshold 524288
+            BypassLockGuard true
+        }
+    }
     headers Authorization
     default_cache_control "public, max-age=86400"
 }
 
 cache @matchdefault {
     ttl 5s
+    badger {
+        path /tmp/badger/default-match
+        configuration {
+            ValueDir default
+            ValueLogFileSize 16777216
+            MemTableSize 4194304
+            ValueThreshold 524288
+            BypassLockGuard true
+        }
+    }
 }
 
 cache @souin-api {}
 ```
 What does these directives mean?  
 |  Key                      |  Description                                                                                                                                 |  Value example                                                                                                          |
 |:--------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------|
+| `allowed_http_verbs`      | The HTTP verbs allowed to be cached                                                                                                          | `GET POST PATCH`<br/><br/>`(default: GET HEAD)`                                                                         |
```

Comment:
Can you elaborate on this? IMHO we should stick with the spec.

Reply:
It's to support the GraphQL partial cache introduced in the previous Souin version. IMHO we should not put aside the GraphQL cache support in the cache-handler.

Reply:
It was the first thing we thought before introduce this but after discussions with some users, they have the use case to restrict the GraphQL cache to POST and PATCH only and we cannot tell to the end user "if you want to enable the GraphQL cache, you have to send only a POST/PATCH request".